autotx/utils/agent/build_goal.py

```diff
@@ -66,35 +66,61 @@ def build_goal(prompt: str, agents_information: str, smart_account_address: ETHA
 def analyze_user_prompt(chat_history: str, agents_information: str, smart_account_address: ETHAddress) -> DefineGoalResponse:
     template = dedent(
         """
-        Based on the following chat history between you and the user: 
-        ```
-        {chat_history}
-        ```
-            
+        You are a task assignment expert. You are good at analyzing the user's chat records to complete the task goals and conditions represented in the user's chat records.
```

Comment:
Should we move this to the persona prompt at the top, or is this placed here strategically?

Reply:
Are you referring to the chat_history variable? This variable is at the end of this prompt

Reply:
I mean't the ` You are a task assignment expert...` part
We have a persona prompt at the top: `You are an AI assistant...`

Reply:
I understand what you mean. I think it can indeed be placed in the system prompt, but my experience with local llm is that the system prompt may not be as effective as directly in the user, so I emphasize it again.